forge-gui/res/cardsfolder/s/serene_sunset.txt

```diff
@@ -1,7 +1,8 @@
 Name:Serene Sunset
 ManaCost:X G
 Types:Instant
-A:SP$ Pump | Cost$ X G | ValidTgts$ Creature | TgtPrompt$ Select X target creatures | TargetMin$ X | TargetMax$ X | KW$ Prevent all combat damage that would be dealt by CARDNAME. | IsCurse$ True | StackDescription$ Prevent all combat damage {c:Targeted} would deal this turn. | SpellDescription$ Prevent all combat damage X target creatures would deal this turn.
+A:SP$ Effect | Cost$ X G | ValidTgts$ Creature | TgtPrompt$ Select X target creatures | TargetMin$ X | TargetMax$ X | ReplacementEffects$ RPrevent | RememberObjects$ Targeted | ForgetOnMoved$ Battlefield | StackDescription$ Prevent all combat damage {c:Targeted} would deal this turn. | SpellDescription$ Prevent all combat damage X target creatures would deal this turn.
```

Comment:
Probably remove StackDescription$

In line 5, probably take `X` out of Description?